src/diffusers/utils/export_utils.py

```diff
@@ -125,7 +125,10 @@ def export_to_video(
     if output_video_path is None:
         output_video_path = tempfile.NamedTemporaryFile(suffix=".mp4").name
 
-    if isinstance(video_frames[0], PIL.Image.Image):
+    if isinstance(video_frames[0], np.ndarray):
```

Comment:
Can't we use the `numpy_to_pil()` method from `processor`?

Reply:
We would need to pass in the processor instance into `export_to_video` though?

Reply:
Ah yes. I am okay as is then.